lib/model.js

```diff
@@ -465,6 +472,18 @@ function filterable(props, subfilters) {
         data = data.split(',');
       }
 
+      if (filter_func === 'within') {
```

Comment:
I think you can leverage the existing condition for 'in' and 'nin'

i.e.
```
if (filter_func === 'in' || filter_func === 'nin' || filter_func === 'within') {
  data = data.split(',');
}
```

Actually, now that I look at this... this should all be done in `coerceData`, so `coerceData` would look like 

```
if (filter_func === 'in' || filter_func === 'nin' || filter_func === 'within') {
  data = data.split(',');
}
if (filter_func === 'within') {
  data = _.map(data, function(item) { parseFloat(item) });
}
if (data && data.toLowerCase && data.toLowerCase() === 'true') {
  ...
} ...
```

I hope that makes sense. That way, the preprocessing on input happens in one place